specification/src/chapter2.adoc

```diff
@@ -79,41 +64,54 @@ Depending on use case and ecosystem requirements, a RoT can be:
 * A dedicated trusted subsystem (HW RoT), supporting a FW RoT
 
 Using a HW RoT moves critical functions and assets off a Hart to a dedicated
-trusted subsystem, which can provide stronger protection against physical or
-logical attack than a complex Hart.
+trusted subsystem, which can provide stronger protection against physical and logical attack than a complex Hart.
 
 [width=100%]
 [%header, cols="5,20"]
 |===
 | ID#
 | Requirement
 
+| CAT_NNN
+| A secure system SHOULD implement a HW RoT
+
 | CAT_NNN
 | A FW RoT SHOULD be the only software in machine mode (M).
```

Comment:
```suggestion
| Software in machine mode (M) SHOULD be measured by the HW RoT e.g. FW RoT, Security services for TEE etc. This software SHOULD allow for updates under control of the HW RoT.
```

Reply:
This is too much information at this point, with concepts that have not been introduced yet. There are separate sections spelling out rules for software authorization, updates, etc.

Also, not all systems will have, and not all systems will need, a HW RoT. Obviously all CoVE systems will, and all GP TEE systems too. But not ALL RISC-V systems will need one.

I have reworked this section a bit to clarify.